src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/test_storage_validators.py

```diff
@@ -116,5 +116,59 @@ def test_storage_get_char_options_validator(self):
         self.assertEqual(result, set('ab'))
 
 
+@api_version_constraint(resource_type=ResourceType.MGMT_STORAGE, min_api='2016-12-01')
```

Comment:
Take another look at the code and my assessment is that the current tests are sufficient:
1. The `--encryption-services` (as the previous `--encryption` option) exists after `2016-12-01` so it is covered.
2. The issue #4220 describes an issue not in the old profile but latest profile.